docs/track_types.rst

```diff
@@ -46,7 +46,8 @@ column" section).
 Intervals can visually encode information using the following three ``options``:
 
 - **colorEncoding**: int or null
```

Comment:
int, 'itemRgb' or null

Reply:
Also, update the `valueColumn` description to:
```
**valueColumn**: int or null
    Use one of the bed columns (1-based) to scale the annotations along the y axis. (default `null`)
```
`none` does not work.

Reply:
👍 Good catch!